Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/CompositingCoordinator.cpp

```diff
@@ -61,8 +65,19 @@ CompositingCoordinator::CompositingCoordinator(WebPage& page, CompositingCoordin
     , m_client(client)
 #if USE(CAIRO)
     , m_paintingEngine(Nicosia::PaintingEngine::create())
+#elif USE(SKIA)
+    , m_skiaAcceleratedBufferPool(nullptr)
+    , m_skiaUnacceleratedThreadedRenderingPool(nullptr)
 #endif
 {
+#if USE(SKIA)
+    if (SkiaRenderingSettings::forceUseCpuRendering()) {
+        if (auto numberOfCpuPaintingThreads = SkiaRenderingSettings::numberOfCpuPaintingThreads(); numberOfCpuPaintingThreads > 0)
+            m_skiaUnacceleratedThreadedRenderingPool = makeUnique<SkiaUnacceleratedThreadedRenderingPool>(numberOfCpuPaintingThreads);
```

Comment:
m_skiaUnacceleratedThreadedRenderingPool should be set to nullptr in invalidate() too, to ensure all pending jobs are cancelled.

Reply:
Correct, fixed.